src/Command/FeedListCommand.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace App\Command;
+
+use App\Repository\FeedRepository;
+use App\Services\Feeds\Mapper\FeedConfigurationMapper;
+use CuyZ\Valinor\Mapper\MappingError;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+#[AsCommand(
+    name: 'app:feed:list',
+    description: 'List feeds from the database',
+)]
+final class FeedListCommand extends Command
+{
+    public function __construct(
+        private readonly FeedConfigurationMapper $configurationMapper,
+        private readonly FeedRepository $feedRepository,
+    ) {
+        parent::__construct();
+    }
+
+    protected function configure(): void
+    {
+        // @todo: Add option to limit based on organization (data available in other PR currently).
+        $this->addOption('show-disabled', '', InputOption::VALUE_NONE, 'Also list disabled feeds');
```

Comment:
`include-disabled` may be a better option name. Setting the flag will show _all_ feeds (including disabled ones) and not just disabled feeds.

Adding a `status` option with values `enabled` (default), `disabled`,  and `all`, say, might be a better approach.

Reply:
Do you known if Symfony command supports an "select list", so it automatically only accepts these value?

Reply:
Fixed